src/test/java/com/faforever/client/login/OAuthEndpointValuesReceiverTest.java

```diff
@@ -51,7 +51,7 @@
 
   @BeforeEach
   void setUp() {
-    clientProperties.getOauth().setTimeoutMilliseconds(1000);
+    clientProperties.getOauth().setTimeoutMilliseconds(0);
```

Comment:
This will cause all the tests to fail as it doesn't get the values instantaneously

Reply:
ok